doc/userguide/devguide/codebase/contributing/github-pr-workflow.rst

```diff
@@ -0,0 +1,36 @@
+GitHub PRs workflow
+===================
+
+Draft PRs
+~~~~~~~~~
+
+A PR should be marked as `draft` if it is not intended to be merged as is, but is waiting for some sort of feedback.
+The author of the PR should explicit what kind of feedback is expected (CI/QA run, discussion on the code, etc...)
+
+GitHub filter is `is:pr is:open draft:true sort:updated-asc`
+
+A draft may be closed if it has not been updated in the last month.
+
+Mergeable PRs
+~~~~~~~~~~~~~
+
+When a Pull Request is intended to be merged as is, the workflow is the following :
+1. get reviewed, and either request changes or get approved
+2. if approved, get staged in a next branch (with other PRs), wait for CI validation (and eventually request changes if CI finds anything)
+3. get merged and closed
```

Comment:
These require indentation to be rendered as a list, with one item per line

Reply:
So, adding a space at the beginning of each line, right ?

Reply:
I usually add a couple, to be safe. Their documentation doesn't mention, this, to be honest, but at least locally I always need that to prevent them from being rendered as simple inline text.

https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html